torch/distributed/rpc/constants.py

```diff
@@ -13,7 +12,8 @@
 
 # For ProcessGroupAgent.
 DEFAULT_NUM_SEND_RECV_THREADS = _DEFAULT_NUM_SEND_RECV_THREADS
-# Same default timeout as in c10d.
-DEFAULT_PROCESS_GROUP_TIMEOUT = default_pg_timeout
+# Ensure that we don't time out when there are long periods of time without
+# any operations against the underlying ProcessGroup.
+DEFAULT_PROCESS_GROUP_TIMEOUT = timedelta(milliseconds=(2**44) - 1)
```

Comment:
curious, any reason for using 2^44 - 1 as the default value?

Reply:
It's the max of a 45 bit signed integer, because https://en.cppreference.com/w/cpp/chrono/duration says that milliseconds is 45 bits wide. But this has been changed in the latest version of the PR.